lib/_http_server.js

```diff
@@ -344,7 +350,13 @@ function connectionListenerInternal(server, socket) {
   socket.on('timeout', socketOnTimeout);
 
   var parser = parsers.alloc();
-  parser.reinitialize(HTTPParser.REQUEST, parser[is_reused_symbol]);
+
+  // Force reinitilalization to make sure AsyncReset is called
+  parser.reinitialize(
+    HTTPParser.REQUEST,
+    true,
```

Comment:
I think we should remove this parameter, or is it used somewhere else?

Reply:
Removed parameter and renamed the method.